Loader/Loader/Loader.server.lua

```diff
@@ -9,88 +9,143 @@
 
 
 ----------------------------------------------------------------------------------------
---                                  Adonis Loader                                     --
+--					Adonis Loader					--
 ----------------------------------------------------------------------------------------
---		   	  Epix Incorporated. Not Everything is so Black and White.		   		  --
+--		Epix Incorporated. Not Everything is so Black and White.		--
 ----------------------------------------------------------------------------------------
---	    Edit settings in-game or using the settings module in the Config folder	      --
+--		Edit settings in-game or using the settings module in the Config folder	--
 ----------------------------------------------------------------------------------------
---	                  This is not designed to work in solo mode                       --
+--		This is not designed to work in solo mode.				--
 ----------------------------------------------------------------------------------------
 
-warn(":: Adonis :: Loading...");
+local function outputMessage(outputType, outputMessage)
+    return warn(string.format(":: Adonis ServerLoader :: %s: %s"), outputType, outputMessage)
+end
```

Comment:
Replacing warn is fine. I don't think a new function outputMessage should be added
You should instead do
`
local OldWarn = warn
local warn = function(...) OldWarn(:: Adonis ServerLoader:: WARN: ", ...) end
`

Reply:
Adonis already replaces warn and print